tools/Google.Cloud.Tools.SbomGenerator/NuspecPackage.cs

```diff
@@ -38,5 +38,11 @@ public class NuspecMetadata
 
         [XmlElement("description")]
         public string Description { get; set; }
+
+        [XmlElement("projectUrl")]
+        public string ProjectUrl { get; set; }
+
+        [XmlElement("license")]
+        public string License { get; set; }
```

Comment:
This is assuming that the license type is "expression". Unfortunately I suspect that with XML serialization, that's relatively hard to validate...

Reply:
Yes, I did look into it briefly, but couldn't find an evident way to do this.